controller/test-files/event/list/ok-area.res.payload.golden.json

```diff
@@ -3,29 +3,21 @@
     {
       "attributes": {
         "name": "system.area",
-        "newValue": null,
-        "oldValue": null,
-        "timestamp": "0001-01-01T00:00:00Z"
+        "timestamp": "0001-01-01T00:00:00Z",
+        "workItemTypeID": "00000000-0000-0000-0000-000000000001"
       },
-      "id": "00000000-0000-0000-0000-000000000001",
+      "id": "00000000-0000-0000-0000-000000000002",
       "relationships": {
         "modifier": {
           "data": {
-            "id": "00000000-0000-0000-0000-000000000002",
-            "links": {
-              "related": "http:///api/users/00000000-0000-0000-0000-000000000002"
```

Comment:
Please note that JSONAPI doesn't allow both to be specified: type+id and links. Instead only one is allowed.